### PR TITLE
Fixed description of a command that was copy/pasted from deletetree

### PR DESCRIPTION
**Pull request by pgporada**

### CODE REVIEW

**Comment by pgporada** [comment]
I signed it!

File: cmd/get_tree_public_key/main.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 
 var (
 	adminServerAddr = flag.String("admin_server", "", "Address of the gRPC Trillian Admin Server (host:port)")
-	logID           = flag.Int64("log_id", 0, "Trillian LogID to delete")
+	logID           = flag.Int64("log_id", 0, "Trillian LogID to retrieve tree public key from")
```
Comment (AlCutter) [inline cmd/get_tree_public_key/main.go]: Maybe change "from" to "for"?

Comment (AlCutter) [reply cmd/get_tree_public_key/main.go]: (or maybe "Trillian LogID whose public key should be fetched"?)
dunno :)

Comment (pgporada) [reply cmd/get_tree_public_key/main.go]: Fixed!

**Pull request merged**